Makefile.dune

```diff
@@ -46,7 +46,7 @@ voboot:
 	dune exec ./tools/coq_dune.exe $(BUILD_CONTEXT)/.vfiles.d
 
 states: voboot
-	dune build $(DUNEOPT) theories/Init/Prelude.vo
+	dune build --display=short $(DUNEOPT) dev/shim/coqtop-prelude
```

Comment:
Is this `--display=short` on purpose or for debugging?

Reply:
A few devs told me in the last meeting that for states they'd like to get some more informative output by default; so I did this tweak; indeed it makes sense as a UI issue IMHO, but we can do as we want.

Reply:
This is configurable on a per user basis by the way. `$ man dune-config` for details. I'd encourage people to just set the output they like there because it's likely they're working on other OCaml projects as well.

Reply:
User config is already documented in the Coq Dune readme. The choice here is in order to this particular legacy target "states" behaves more similarly for people migrating. This was a FAQ of `state` users: "I want it to display short mode".